### ISSUE TITLE
POST Adopt a Drain Background info

### ISSUE THREAD

**Issue by jeanwalshie** [sfbrigade/adopt-a-drain#190]
hello all!

we at the SFPUC are getting lots of questions from other City agencies about the Adopt a Drain app and program - how it was developed, how much it cost, background, etc. I wrote up this language and i'm hoping someone can post the language somewhere on the page. perhaps it could link to the logo in the upper left corner? and/or maybe we can place a sentence at the bottom, under the logos, that says "Adopt a Drain SF Background - learn more". 
[AdoptaDrainsummary.docx](https://github.com/sfbrigade/adopt-a-drain/files/470346/AdoptaDrainsummary.docx)

The text is below (without hyperlinks) and attached. Thanks in advance! (also feel free to suggest changes/clarifications to the text.)
 
Introduction
The San Francisco Adopt a Drain program enables San Franciscans to “adopt” one of the city’s 25,000 storm drains or catch basins and pledge to keep it free of leaves and debris. The program helps reduce localized flooding and fosters community engagement by encouraging the public to take an active role in improving their neighborhood. 
 
Partners
In late 2015 several San Francisco City agencies partnered with volunteers from Code for San Francisco to develop the Adopt a Drain SF web application. 

The San Francisco Public Utilities Commission (SFPUC), the agency that manages the sewer system, including 25,000+ catch basins and storm drains, and San Francisco Public Works, the agency that maintains the database of sewer infrastructure assets, worked with the City’s Open Data Program in the Mayor’s Office to get the catch basin location data uploaded to the Open Data Portal (SF Open Data).

Code for San Francisco and other volunteers then used a web-based platform called Github to develop the open source web application on Ruby on Rails. Github enabled contributors to work on the project from anywhere in the world. They modeled Adopt a Drain SF on Boston’s Adopt a Hydrant, for which Code for America had provided initial support to develop in 2012.

SFPUC staff got internal buy-in and support for the program, created the graphics for the icons and logo, wrote the copy for the website and emails, reviewed the website for disability access, secured funding to host the site, coordinated with other SF city agencies and the City of Oakland, and promoted the program on social media, traditional media, TV advertising and various other channels. SF Open Data staff initially paid for the site to be hosted and provided ongoing strategic and technical support. Approximately 25 people contributed to coding and developing the app.  

Next Steps 
The SFPUC is planning to expand the program to offer free training, tools, and supplies to drain adopters.

**Comment by jeanwalshie** [comment]
@jasonlally @hackajesse @jszwedko -- thoughts on this? we are planning a big cable TV advertising campaign this fall, and would love to get this text updated before then.

**Comment by jszwedko** [comment]
@jeanwalshie yep! I can add that today.

**Comment by jszwedko** [comment]
@jeanwalshie I added the content as a modal dialog and linked it from the footer and the header -- let me know what you think!

**Comment by jeanwalshie** [comment]
@jszwedko thanks!  -- but i don't see it! :(

**Comment by jszwedko** [comment]
@jeanwalshie I forgot we have a staging server :) -- do you want to verify it there before I push it up to production?. The URL is https://adoptadrainsf-staging.herokuapp.com/

**Comment by jeanwalshie** [comment]
oh, i see, thanks! 
ok, now i have a few more thoughts:
1) i dont think we need to say "We're just getting started!" What do you think about removing that line?
2) in the footer we now have "Learn more" and "send feedback". I'm thinking we should remove the "send feedback" link and instead add a line to the bottom of the "learn more" text that says 
Contact
Email questions and feedback to info@sfwater.org

**Comment by jeanwalshie** [comment]
@jszwedko @jasonlally hi guys! if i confused things i'm sorry. i think what you put up on the test site is fine for now -- my comments/ideas can be dealt with at a later date. i'd like to get this content up though.
thank you!

**Comment by jszwedko** [comment]
Hi @jeanwalshie!

Apologies for the delay -- I started to make the revisions during the Code for SF hack night this past Wednesday, but didn't complete them (and was out of town this weekend).

My thoughts:

* I agree and removed the alpha text
* I think we should still leave the feedback link in the sidebar (to lower the barrier to giving feedback -- e.g. if I'm on the page and want to give feedback, it wouldn't be super clear that I have to click "Learn more" first)
* Typically the logo of a web page is a link to the root page (which I've added). Admittedly since this is a single page app, there isn't much value in having a link, but I think that will be the least confusing to users

I pushed up these changes to the staging server. Let me know what you think or if you disagree with any of the above!

If you want the background info to be easier to find, I can think about better ways to make it more visible.
